docs/Database_Schema.md

```diff
@@ -67,9 +67,10 @@ CREATE TABLE IF NOT EXISTS "enrollments" (
 
 | Column            | Data Type | PK | Not NULL | Default  |Description|
 |:----------------- |:--------- |:--:|:--------:| -------- | --------- |
-| block_height      | INTEGER   | Y  | Y        |          | The height of the block|
-| tx_index          | INTEGER   | Y  | Y        |          | The index of transaction in the block |
-| tx_hash           | BLOB      |    | Y        |          | The hash of transaction |
+| tx_hash           | BLOB      | Y  | Y        |          | The hash of transaction |
+| tx_index          | INTEGER   |    | Y        |          | The index of transaction in the block |
+| block_hash        | BLOB      |    | Y        |          | The hash of the current block |
```

Comment:
This column is unnecessary from the db normalization point of view.
The primary key of the block is still defined by `height`.
If you want to use this as an FK, you must change the PK in the block table to Block_hash.

Reply:
That's a good point.
If I modify this, the changes are too big, so I will create a new issue and proceed.
I will remove the commit from the current PR.